pkg/util/file.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package util
+
+import "os"
+
+// FileExists returns True if the supplied file path exists, false otherwise
+func FileExists(path string) bool {
+	_, err := os.Stat(path)
+	return !os.IsNotExist(err)
```

Comment:
https://golang.org/pkg/os/#IsExist ? lol

Reply:
Not sure if we can replace `!IsNotExist` with `IsExist`. Because `os.IsExist` and `os.IsNotExist` are error checkers (wrappers of `errors.Is` function) - they both return false if the given error is `nil`.

Reply:
Ah I see, they check error message conditions. How confusing. Thanks!